.vsts-ci.yml

```diff
@@ -165,7 +166,20 @@ stages:
             /p:HelixBuild=$(Build.BuildNumber)
             /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/SendToHelix.binlog
             /p:HelixAccessToken=$(HelixApiAccessToken)
-          displayName: Run Helix Tests
+          displayName: Run Helix Unit Tests
+          env:
+            SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+            HelixAccessToken: $(HelixApiAccessToken)
+        - script: ./eng/common/msbuild.sh --warnaserror false --ci
```

Comment:
Why do you need to add another step, Ideally we would run both with just the one call.

Reply:
You can't unless we run it on PIs as well since there is no way specify which project runs on which queue. If I create a target which call sendToHelix twice then I hit this issue: https://github.com/dotnet/arcade/pull/4107

Other thing is that it makes sense to first run unit tests before running any hardware tests so that should be fine